lib/mastodon/rest/accounts.rb

```diff
@@ -12,6 +12,17 @@ def verify_credentials
         perform_request_with_object(:get, '/api/v1/accounts/verify_credentials', {}, Mastodon::Account)
       end
 
+      # Update authenticated account attributes
+      # @param options [Hash]
+      # @option options display_name [String] The name to display in the user's profile
+      # @option options note [String] A new biography for the user
+      # @option options avatar [String] A base64 encoded image to display as the user's avatar
```

Comment:
Despite what the docs say, avatar/header can be a good old form file upload, isn't limited strictly to Base64. The mechanism is no different to the upload media method. This just a comment, this is fine, maybe you'll want to specify it later.